website/aggregated_questions.qmd

```diff
@@ -0,0 +1,251 @@
+
+Will polygenic screening for physical or cognitive traits lead to a class-action lawsuit [scope of lawsuit required] in the United States by 20XX?
+
+Will more than 30 countries [scope of countries required] introduce regulations [scope of regulations required] ensuring equal access [scope of equal access required] to genetic enhancement technologies by 20XX?
+
+Will there be a criminalization of unregulated [attempted topic of question is "black market" or "back market"] genetic modifications in over 20 countries [scope of countries required] by 20XX?
+
+Will a genetically modified individual be legally [scope of legal definition required] denied citizenship or human rights by 20XX?
+
+Will more than XX% of countries adopt ethical guidelines [scope of ethical guidelines required] prohibiting the selection of embryos based on non-health-related traits by 20XX?
+
+Will human genome editing be used as a defense in a court of law in a high-profile case [defense in country and publicity of case definitions required] by 20XX?
+
+Will there be widespread protests / riots [scope of civil disobience surrounding topic required] against the use of polygenic screening in schools or workplaces [OR other settings] by 20XX?
+
+Will any country [some scope correction probably useful here] allow parents to choose the sexual orientation of their children via genetic screening or editing by 20XX?
+
+Will more than XX% of the global population use genetic enhancements [measurement for global population and use of genetic enhancement required] that impact physical appearance by 20XX?
+
+In what year will over XX% of countries have a regulatory framework for genetic editing of embryos [current log of countries with genetic editing frameworks required]?
+
+What percentage of the global [OR country OR country consortium] population will support [definition of support required] the right to use genetic engineering for enhancement by 20XX?
+
+In what year will the first international legal case [more detail on international legal case required] be filed concerning genetic discrimination in employment [OR other area]?
+
+By what year will more than XX% of the world’s population [OR country OR country consortium] have undergone some form of genetic enhancement for non-medical purposes [measurement for this question required]?
+
+In what year will more than XX% of healthcare systems globally [scope of global healthcare systems required] implement genetic screening as a routine part [definition of routine part required] of preventive care?
+
+By what year will more than 30 countries [scope of countries required] have established policies [scope of policies required] to protect against genetic inequality [definition of access required] in access to enhancements?
+
```

Comment:
suggestion: "have established welfare programs and/or hard caps designed to equalize access to genetic enhancements"